Automattic/wp-e2e-tests
# Add Jetpack SSO connect flow

adds SSO connect flow as requested in #777 

To test this specific flow:

0. decrypt updated config. make sure `subscriberUser` is in your confg
1. Set `JETPACKHOST` env var to `PRESSABLE` / `JN`: e.g `export JETPACKHOST=PRESSABLE`
2. run the test: `./node_modules/.bin/mocha ./specs-jetpack-calypso/wp-jetpack-sso-connect-spec.js -g 'Connect via SSO'`
